client/src/components/PageDetails.js

```diff
@@ -43,6 +43,13 @@ export class PageDetails extends React.Component {
             persist_content={true}
           />
         </div>
+        <div className="pagedetails__report-a-problem fcol-md-8 fcol-sm-12">
+          <Details
+            summary_content={text_maker("feedback")}
+            content={<EmailFrontend template_name="feedback" />}
+            persist_content={true}
+          />
+        </div>
```

Comment:
Remember to disable this before merging, we'll stick with the legacy survey page until after the current popup cmpaign ends and the mongo logging's ready.

Reply:
added a radio buttons case to test template 👍

Reply:
and updated for the end to end testing too 👍